menu.js

```diff
@@ -22,17 +22,212 @@ form.addEventListener('submit', (event) => {
+    <div class="button-container">
+        <button class="button-see-project" type="button" id="${projectInfo[i].id}">
+            See project
+        </button>
+    </div>
+</article>
+
+`;
+  card.innerHTML += popupcard;
+  work.appendChild(card);
+});
+
+// function is to create div and append it to body and add class to it
+function makeDiv(event) {
+  const popup = document.createElement('div');
+  document.body.append(popup);
+  popup.classList.add('pop-up-class');
+
+  // add elements
+  const header = document.createElement('div');
+  header.classList.add('header-pp');
+  const projecthead = document.createElement('h2');
+  projecthead.classList.add('project-head');
+  const projectimg = document.createElement('img');
+  projectimg.classList.add('image-pp');
+  const positiondiv = document.createElement('div');
+  positiondiv.classList.add('position-div-pp');
+  const projectdescrip = document.createElement('p');
+  projectdescrip.classList.add('project-descrip');
+  const langbtndiv = document.createElement('div');
+  langbtndiv.classList.add('lang-btn-div');
+  const projecttech = document.createElement('ul');
+  projecttech.classList.add('lang-container-pp');
+  const projectbtncontainer = document.createElement('div');
+  const btnseelive = document.createElement('a');
+  const btnseesource = document.createElement('a');
+  const closeButton = document.createElement('button');
+  closeButton.classList.add('closeButton-class');
+  closeButton.id = 'close-button';
+
+  // append elements inside the div
+  popup.append(
+    header,
+    projecthead,
+    closeButton,
+    projectimg,
+    positiondiv,
+    projectdescrip,
+    langbtndiv,
+    projecttech,
+    projectbtncontainer,
+  );
+  header.append(projecthead, closeButton);
+  positiondiv.append(projectdescrip, langbtndiv, projecttech, projectbtncontainer);
+  langbtndiv.append(projecttech, projectbtncontainer);
+  projectbtncontainer.append(btnseelive, btnseesource);
+
+  // add content to the elements of div
+  const projectId = parseInt(event.target.id, 10);
+  projecthead.textContent = projectInfo[projectId].name;
+  projectimg.src = projectInfo[projectId]['featured image'];
+  projectdescrip.textContent = projectInfo[projectId].description;
+
+  for (let i = 0; i < projectInfo[projectId].technologies.length; i += 1) {
+    const projectLi = document.createElement('li');
+    projectLi.className = 'languages-pp';
+    projectLi.textContent = projectInfo[projectId].technologies[i];
+    projecttech.appendChild(projectLi);
+  }
+
+  projectbtncontainer.classList.add('btn-container-class-pp');
+  btnseelive.href = projectInfo[projectId]['link to live version'];
+  btnseelive.classList.add('btn-seelive-pp');
+  btnseelive.target = '_blank';
+  btnseelive.textContent = 'See Live';
+  btnseesource.href = projectInfo[projectId]['link to source'];
+  btnseesource.classList.add('btn-seesource-pp');
+  btnseesource.target = '_blank';
+  btnseesource.textContent = 'See Source';
```

Comment:
- [ ] OPTIONAL: Please add a distance between both buttons, they are too close and there is not look well. I know you can do it, just try please 💯 😊

Reply:
Thank you so much for approving my project. Dear Kandy Peter.